Fix links and add two last awards in About page (#161)
openfisca/openfisca.org: content/about.fr.md

En 2018, le responsable produit qui avait dirigé les efforts d'internationalisation [est parti](https://www.digital.govt.nz/blog/labplus-lessons-from-matti-schneider-about-service-innovation-at-french-state-incubator-beta-gouv-fr/) en Nouvelle-Zélande, où le [Service Innovation Lab](https://serviceinnovationlab.github.io/projects/legislation-as-code/) récemment créé terminait la rédaction de son rapport [Better Rules](https://www.digital.govt.nz/blog/what-is-better-rules/). Cet élément théorique fondamental correspondait parfaitement à la mise en œuvre par OpenFisca de la « transformation de la loi en code », ce qui aboutit à l'application _Rates Rebate_, le premier service en production en dehors de l'Europe. D'autres zones de la région Pacifique (Australie, Polynésie, Canada) ont été progressivement exposées à OpenFisca et l'ont adopté.

> En 2019, OpenFisca remporte le [prix du produit ouvert le plus innovant](https://joinup.ec.europa.eu/collection/sharing-and-reuse-it-solutions/sharing-reuse-awards-2019-results#oss-inno) du programme Joinup de la Commission Européenne.   

Reviewer: ```suggestion
En 2019, OpenFisca est reconnu comme [logiciel libre le plus innovant](https://joinup.ec.europa.eu/collection/sharing-and-reuse-it-solutions/sharing-reuse-awards-2019-results#oss-inno) par la Commission Européenne à travers son programme Joinup.
```

> Cette même année, le premier simulateur [LexImpact](https://leximpact.an.fr) destiné aux parelementaires naît. Cet outil d'étude d'impact basé sur OpenFisca avec l'Assemblée nationale, leur permet d'évaluer l'impact de réformes sur des cas types ou l'ensemble de la population française en quelques secondes.

Reviewer: ```suggestion
Cette même année, le premier simulateur [LexImpact](https://leximpact.an.fr) destiné aux parlementaires naît. Cet outil d'étude d'impact fourni par l'Assemblée nationale leur permet d'évaluer l'impact de réformes sur des cas types ou l'ensemble de la population française en quelques secondes.
```

En 2020, compte tenu de l'adoption croissante dans les administrations et de la masse critique de contributeurs, la DINUM a décidé de cesser de financer les efforts de croissance et de maintenance d'OpenFisca, suggérant que ceux-ci soient financés collectivement par les réutilisateurs. Une équipe cœur minimale a été maintenue grâce à un financement de l'Agence nationale de la cohésion des territoires (ANCT), mais la croissance et les améliorations techniques ont considérablement ralenti en l'absence de modèle économique clair.

En 2021, des membres de la société civile ont commencé à modéliser la législation britannique, puis américaine, ce qui a conduit à la création de [PolicyEngine](https://policyengine.org).

En 2022, alors que l'utilisation augmentait dans plusieurs pays sans contribution claire à un moteur de plus en plus obsolète, les moyens d'un fonctionnement durable d'OpenFisca indépendamment de tout État ont été évalués par l'équipe centrale sur les fonds ANCT.

> En 2023, la communauté OpenFisca reçoit le prix des [Créations gouvernementales innovantes](https://www.worldgovernmentsummit.org/awards/edge-of-government) au Sommet Mondial des Governements.

Reviewer: ```suggestion
En 2023, la communauté OpenFisca est sélectionnée parmi plus de 1 000 candidats sélectionnés par l'OCDE et reçoit le prix d'innovation [Edge of Government](https://www.worldgovernmentsummit.org/awards/edge-of-government) au World Government Summit.
```

Author: Thx for the improvement! Keeping the suggestion but imho, `Edge of Government` is less clear than `Créations gouvernementales innovantes` (that was translated from the arabic name [ابتكارات الحكومات الخلاقة](https://www.worldgovernmentsummit.org/ar/awards/edge-of-government)).

Reviewer: It is less _clear_, but no one else in the world would use this literal translation from Arabic except us 😉
